### PR TITLE
Tilføj hele frontenden :D

### PR DESCRIPTION
**Pull request by ltranu20**
squash and merge plz

### CODE REVIEW

File: V8-R8-Hub/Pages/Admin/Game.cshtml.cs [hunk 1]
```diff
@@ -22,7 +22,10 @@ public class GameModel : PageModel {
 		public async Task OnGetAsync(Guid guid) {
 			Guid = guid;
 			await Task.WhenAll(
-				Task.Run(async () => GameBrief = await _gameService.GetGame(guid)),
+				Task.Run(async () => {
```
Comment (JE-FH) [inline V8-R8-Hub/Pages/Admin/Game.cshtml.cs:25]: fix dencker pls

File: V8-R8-Hub/Services/GameAssetService.cs [hunk 2]
```diff
@@ -116,13 +116,16 @@ SELECT ga.path
...
 				"image/png",
 				"image/jpeg",
 				"image/gif",
-				"image/tiff"
-			});
+				"image/tiff",
+                "audio/wav",
+                "application/x-javascript", // safari uploader .js i det her format
+                "audio/x-wav", // safari uploader .wav i det her format
+            });
```
Comment (JE-FH) [inline V8-R8-Hub/Services/GameAssetService.cs:128]: fix indentation

**Review by JE-FH — CHANGES_REQUESTED** [review]
Yes yes bare fix comments

File: V8-R8-Hub/Services/GameAssetService.cs (hunk 2 above)
Comment (Djepse20) [reply V8-R8-Hub/Services/GameAssetService.cs]: should be fixed.

**Review by JE-FH — APPROVED** [review]

**Comment by ltranu20** [comment]
maler dir_214

**Pull request merged**